src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -840,6 +840,35 @@ def list_publish_profiles(resource_group_name, name, slot=None):
     return converted
 
 
+def enable_cd(resource_group_name, name, enable, slot=None):
+    settings = []
+    settings.append("DOCKER_ENABLE_CI=" + enable)
+    update_app_settings(resource_group_name, name, settings, slot)
+    if enable == 'true':
```

Comment:
Suggest remove the condition, so the command can be consistent on output. The format I would suggest
```json
{
   "DOCKER_ENABLE_CI": "true or false"
   "CD_URL" : "<url or empty>" 
}
```